### PR TITLE
Harolddavis/ch102866/create a readme md for fruit shoppe app

### PR DESCRIPTION
**Pull request by HaroldFS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+
+## Using the app
+
+The Shoppe is a super simple e-commerce application. Build and run the app on your emulator, you can:
+
+- Browse a list of products int the _Market_.
+- Use the **Add to Cart** button to add products to your shopping cart.
+- Go to your _Shopping Cart_ by clicking on the cart icon on the top right corner.
+- Review your cart and then click the **Checkout** button.
+- Fill out the form on the _Checkout_ view and click **Purchase**. The app persists locally all the information from this view. Please do **not** use your real information
```
Comment (van-fs) [inline README.md]: Missing period at end.

Comment (van-fs) [reply README.md]: Does the app also have exclusion rules if the end user did try to enter sensitive information?  If so, we might want to re-iterate that point on what PII is/isn't recorded in sessions.

**Review by van-fs — APPROVED** [review]
LGTM, see my two comments for the README.

File: README.md (hunk 1 above)
Comment (HaroldFS) [reply README.md]: Interesting, well at the moment I have not built this out but I did not plan on recording this info, I will bring it up again once the checkout flow is complete.

**Pull request merged**